risks/secure-communication/secure-communication.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"github.com/threagile/threagile/model"
+)
+
+type secureCommunication string
+
+var CustomRiskRule secureCommunication
+
+func (r secureCommunication) Category() model.RiskCategory {
+	return model.RiskCategory{
+		Id:                         "insecure-communication",
+		Title:                      "Use Of Weak Cryptography in transit",
+		Description:                "To ensure confidentiality during transit strong encryption must be used; weak, broken or soon to be deprecated algorithms must be avoided and recommended key lengths must be applied.",
+		Impact:                     "Weak cryptography can result in information disclosure and a false sense of security.",
+		ASVS:                       "v4.0.3-9.X - Communication",
+		CheatSheet:                 "https://cheatsheetseries.owasp.org/cheatsheets/Transport_Layer_Protection_Cheat_Sheet.html",
+		Action:                     "Cryptography",
+		Mitigation:                 "Stay current with recommended industry advice on secure configuration of TLS, or similar.",
+		Check:                      "Referenced ASVS chapters and cheat sheets",
+		Function:                   model.Operations,
+		STRIDE:                     model.InformationDisclosure,
+		DetectionLogic:             "Encrypted communication links",
+		RiskAssessment:             "Risk is based on the confidentiality score of data sent or recieved.",
+		FalsePositives:             "None",
+		ModelFailurePossibleReason: false,
+		CWE:                        327,
+	}
+}
+func (r secureCommunication) SupportedTags() []string {
+	return []string{}
+}
+func (r secureCommunication) GenerateRisks() []model.Risk {
+	risks := make([]model.Risk, 0)
+	for _, comm := range model.CommunicationLinks {
+		if comm.Protocol.IsEncrypted() {
```

Comment:
Pull commlinks per technical asset, filter out the encrypted ones and pick the most critical.